index.html

```diff
@@ -9,14 +9,13 @@
     <title>Document</title>
 </head>
 <body>
-    <div class="header">
+    <header >
         <div class="header_elements name"><h3>ikpro</h3></div>
-        
-        <div class="header_elements icons"><i class="fa-solid fa-bars"></i></div>
-    </div>
+        <div class="header_elements icons"><a href=#><i class="fa-solid fa-bars"></i></a></div>
+    </header>
 
-    <div class="intro">
-        <div><h1>Inayat-khan</h1></div>
+    <section class="intro">
+        <section><h1>Inayat-khan</h1></section>
```

Comment:
### OPTIONAL
- [ ] Please remove this `section` tag here and allow just the `h1`